service/service.go

```diff
@@ -0,0 +1,190 @@
+package service
+
+import (
+	"context"
+	"github.com/ONSdigital/dp-search-api/api"
+	"github.com/ONSdigital/dp-search-api/elasticsearch"
+	"github.com/ONSdigital/dp-search-api/query"
+	"github.com/ONSdigital/dp-search-api/transformer"
+	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
+	"os"
+
+	esauth "github.com/ONSdigital/dp-elasticsearch/v2/awsauth"
+	elastic "github.com/ONSdigital/dp-elasticsearch/v2/elasticsearch"
+	dphttp "github.com/ONSdigital/dp-net/http"
+	"github.com/ONSdigital/dp-search-api/config"
+	"github.com/ONSdigital/log.go/v2/log"
+)
+
+type Service struct {
+	config              *config.Configuration
+	healthCheck         HealthChecker
+	api                 *api.SearchAPI
+	router              *mux.Router
+	server              HTTPServer
+	serviceList         *ExternalServiceList
+	esSigner            *esauth.Signer
+	elasticHTTPClient   dphttp.Clienter
+	queryBuilder        api.QueryBuilder
+	elasticSearchClient elasticsearch.Client
+	transformer         transformer.Transformer
+}
+
+//New creates a new service
```

Comment:
we tend to keep a space between `//` and description in comment
```suggestion
// New creates a new service
```

Reply:
Please check the other methods in this file